README.md

```diff
@@ -616,9 +616,4 @@ For example, `shopify_test_redis_master` or `shopify_development_mysql_1`.
 
 ### Release
 
-1. Update `CHANGELOG.md`
-1. Change versions in `README.md`
-1. Create a release commit and a signed tag.
-1. [Bump version for Homebrew](https://github.com/Shopify/homebrew-shopify/blob/master/toxiproxy.rb#L9)
-
-[blog]: https://shopifyengineering.myshopify.com/blogs/engineering/building-and-testing-resilient-ruby-on-rails-applications
+See [RELEASE.md](./RELEASE.md)
```

Comment:
```suggestion
See [RELEASE.md](./RELEASE.md)

[blog]: https://shopifyengineering.myshopify.com/blogs/engineering/building-and-testing-resilient-ruby-on-rails-applications
```

Reply:
I think we can remove this section at all.